packages/forms-web-app/src/controllers/selected-appeal/questionnaire-details/appeal-process-details-rows.js

```diff
@@ -0,0 +1,30 @@
+const { formatProcedurePreference, formatYesOrNo, formatConditions } = require('@pins/common');
+
+/**
+ * @param {import('appeals-service-api').Api.AppealCaseWithAppellant } caseData
+ * @returns {import("@pins/common/src/view-model-maps/rows/def").Rows}
+ */
+exports.appealProcessRows = (caseData) => {
```

Comment:
since you're creating this array by passing in `caseData` is there any reason not to run all the conditions up front so they're already booleans when you filter them?

Reply:
not quite sure what you mean by this?

Reply:
I have left this as it was due the same reason above

Reply:
You don't need to make the logic change in `displayRows`, you can do that as
```
condition !== undefined && condition !== null
```
The key is you don't need to pass the data in again

Reply:
Ah ok, I cant just put `condition: caseData.lpaProcedurePreference` as the condition expects `() =>`  so I can do `() => caseData.lpaProcedurePreference` as this will use the data that is being passed as the argument to appealProcessRows rather than every row each being passed the same new data. 

I could change the row definition but this would affect the implementation in the selected appeal and appeal details pages